xm-commons-mail/src/main/java/com/icthh/xm/commons/mail/provider/communication/CommunicationMailService.java

```diff
@@ -0,0 +1,47 @@
+package com.icthh.xm.commons.mail.provider.communication;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.icthh.xm.commons.messaging.communication.CommunicationMessage;
+import com.icthh.xm.commons.tenant.TenantContextHolder;
+import com.icthh.xm.commons.topic.service.KafkaTemplateService;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CommunicationMailService {
```

Comment:
I think better to remove work Mail and move to communication commons?

Reply:
Done